Restore GCC 14 compat flags for net-snmp 5.9.1 (upstream kiddin9 dropped them;
build fails with implicit-function-declaration / incompatible-pointer-types).

diff --git a/net-snmp-ssl/Makefile b/net-snmp-ssl/Makefile
index f4219f5d4..794aab8f9 100644
--- a/net-snmp-ssl/Makefile
+++ b/net-snmp-ssl/Makefile
@@ -182,7 +182,7 @@ SNMP_TRANSPORTS_INCLUDED = Callback UDP Unix
 
 SNMP_TRANSPORTS_EXCLUDED = TCP TCPIPv6
 
-TARGET_CFLAGS += $(FPIC)
+TARGET_CFLAGS += $(FPIC) -std=gnu17 -Wno-error=incompatible-pointer-types -Wno-error=implicit-function-declaration
 TARGET_CPPFLAGS += -I$(STAGING_DIR)/usr/include/libnl-tiny
 
 CONFIGURE_ARGS += \
